server/views/content/index.ejs

```diff
@@ -1,15 +1,29 @@
 <% include ../partials/header.ejs %>
 
     <!-- MAIN CONTENT -->
-    <div class="container">
+    <div style="background-color:burlywood;min-height: 400px;width: 100%;">
```

Comment:
Why remove container class? I think this is why the vertical scroll is showing

Reply:
I deleted because It didn't cover 100% of the page but now I fixed it.